aviary/mission/gasp_based/ode/unsteady_solved/test/test_unsteady_flight_conditions.py

```diff
@@ -73,7 +73,7 @@ def _test_unsteady_flight_conditions(self, ground_roll=False, input_speed_type=S
         assert_near_equal(eas, tas * np.sqrt(rho / rho_sl))
         assert_near_equal(dTAS_dt_approx, np.zeros(nn))
 
-        p.run_model()
+        # p.run_model()  # why run twice?
```

Comment:
This was probably just a cut and paste error.

Reply:
Agree. I removed it now.